src/main/java/pl/rspective/voucherify/client/api/VoucherifyApi.java

```diff
@@ -140,4 +142,44 @@
     @POST("/redemptions/{id}/rollback")
     VoucherRedemptionResult rollbackRedemption(@Path("id") String redemptionId, @Query("tracking_id") String trackingId, @Query("reason") String reason);
 
+    /**
+     * Create a customer using given Customer
+     *
+     * @param customer (required)
+     *
+     * @return created User
+     */
+    @POST("/customers")
+    Customer createCustomer(@Body Customer customer);
+
+    /**
+     * Fetch a Customer using Customer ID
+     *
+     * @param customerId (required)
+     *
+     * @return customer with given ID
+     */
+    @GET("/customers/{id}")
+    Customer fetchCustomer(@Path("id") String customerId);
```

Comment:
@imvanzen I know that in voucher we have `fetch` but let's change to name `getCustomer`. Thanks